app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -307,12 +307,15 @@ def availability_zones
 
   def snapshots
     collector.snapshots.each do |snapshot|
+      # API not returning size of snapshot, thus have to tally up size of associated volumes
```

Comment:
Just a minor change to the wording here maybe,

```suggestion
      # API does not return the total size of snapshot so we have to sum the size of the associated volumes
```

Reply:
Updated in the latest force-push.